### DISCUSSION TITLE
Controller -> Send HA Discovery

### DISCUSSION THREAD

**Discussion by MichaelSordje** [goecharger/go-eController-API#9] [Ideas]
a nice new feature  would be a better home assistant integration.

To integrate the controller as a MQTT Device for Home Assistant, it would be cool, if the Controller can send a Discovery for home assistant.
Otherwise we have to get each Value per mqtt or REST and publish a sensor in configuration.yaml

After that i have all entities in the MQTT Device
![image](https://github.com/goecharger/go-eController-API/assets/8821876/079de9f2-77dc-4687-ba0e-511465099cea)

for example like watermeter or nuki:
![image](https://github.com/goecharger/go-eController-API/assets/8821876/07b3cd05-6617-4a83-bc02-650765e5a1e3)

**Comment by CommanderRedYT** [comment]
Can you update to the newest firmware (1.0.9) and check if a key called "mhe" exists? If so, configure your Mqtt server and set it to true
